pkg/metaclient/common.go

```diff
@@ -0,0 +1,167 @@
+package metaclient
+
+import "fmt"
+
+// ResponseHeader is common response header
+type ResponseHeader struct {
+	// ClusterId is the ID of the cluster which sent the response.
+	// Framework will generate uuid for every newcoming metastore
+	ClusterID string
+}
+
+func (h *ResponseHeader) String() string {
+	return fmt.Sprintf("clusterID:%s;", h.ClusterID)
+}
+
+// Put Response
+type PutResponse struct {
+	Header *ResponseHeader
+}
```

Comment:
If `revision` is used, `PutResponse` should contain `revision` too. Such as the usage
1. put a key for the first time, return the `revision` such as `1000` of this key
2. put the key for another time, with `revision = 1000` to ensure there is concurrent update of this key between step-1 and step-2

Reply:
Seems that it's easy to get the revision for putResponse when backend is etcd.
But when the backend is TiDB/MySQL, we still need another get to fetch the latest revision.